### PR TITLE
[fluidlite] Add new port

### PR DESCRIPTION
**Pull request by Ghabry**
- What does your PR fix?

It adds a new library "fluidlite".

Biggest advantage: _No dependency on glib_. Actually no single dependency. Perfect when you just want to work with Soundfonts.

Fluidlite has a API compatible to Fluidsynth but it lacks some features: No built-in audio player, no MIDI file parser. So the main purpose is sampling audio provided by an external midi sequencer library.

- Which triplets are supported/not supported? Have you updated the CI baseline?

I tested x86-windows, x64-windows and x64-windows-static.

The CMakeLists builds on Linux and macOS, but not tested through vcpkg.

No idea what a CI baseline is.

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?

I think so.

I have a question:
The library has no versions, I just used the latest master commit as a version. Is this okay or how is the procedure?

### CODE REVIEW

File: ports/fluidlite/CONTROL [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Source: fluidlite
+Version: 813e548
```
Comment (NancyLi1013) [inline ports/fluidlite/CONTROL:2]:
```suggestion
Version: 2020-08-20
```
The version should be a date that you accept the commit.

Please see https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/control-files.md#version

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Comment by NancyLi1013** [comment]
Hi @Ghabry 
Thanks for this PR.
There are some regressions on `x64-uwp` and `arm-uwp`. Could you please look into them?

File: ports/fluidlite/CONTROL (hunk 1 above)
Comment (Ghabry) [reply ports/fluidlite/CONTROL:2]: Oh must have missed that paragraph. Thanks!

**Comment by Ghabry** [comment]
Ignore my "XXX" commits they are just for testing because I can't get UWP installed locally (Internet too slow, Download takes days).
I will try to fix them and then submit a patch upstream. Will tell you when I'm ready :)

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
@Ghabry 
LGTM now. Thanks for your efforts.

**Pull request merged**